src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+
+import { AvatarSt } from './style';
+
+interface AvatarProps {
+  backgroundImage: string;
+}
+
+const Avatar: FC<AvatarProps> = ({ backgroundImage }) => {
+  return <AvatarSt backgroundImage={backgroundImage} />;
+};
+
+export default Avatar;
```

Comment:
1. Опять дублируется интерфейс пропсов для компонента и styled компонента
2. И компонент тоже лишняя обертка тут

Можно все заменить на просто
```suggestion
export default AvatarSt;
```

Reply:
ок